v19.2/vectorized-execution.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Vectorized Query Execution
+summary: The CockroachDB vectorized SQL query execution engine processes query plans using a column-oriented model to improve performance.
+toc: true
+---
+
+ <span class="version-tag">New in v19.2:</span> CockroachDB supports column-oriented ("vectorized") query execution on all [supported SQL operations](https://www.cockroachlabs.com/docs/stable/sql-feature-support.html).
+
+ Many SQL databases, including MySQL, PostgreSQL, and all versions of CockroachDB released prior to 19.2, execute [query plans](https://en.wikipedia.org/wiki/Query_plan) one row of table data at a time. The CockroachDB vectorized execution engine dramatically improves performance over row-oriented execution by processing each component of a query plan on type-specific batches of column data. Column-oriented execution models contrast with row-oriented execution models, which can offer good performance for [online transaction processing (OLTP)](https://en.wikipedia.org/wiki/Online_transaction_processing) queries, but suboptimal performance for [online analytical processing (OLAP)](https://en.wikipedia.org/wiki/Online_analytical_processing) queries.
+
+
+## Configuring vectorized execution
+
+By default, vectorized execution is enabled in CockroachDB for all supported "streaming" SQL operations (i.e. operations that do not require [memory buffering](https://en.wikipedia.org/wiki/Data_buffer)). Non-streaming operations include global [sorts](query-order.html), [window functions](window-functions.html), [hash joins](joins.html#hash-joins), and [merge joins](joins.html#merge-joins) on non-unique columns. Merge joins on columns that are guaranteed to have one row per value, also known as "key columns", are considered streaming operations.
```

Comment:
I'm glad we are linking to the actual things here (joins, etc.)  I think we should go further and be very explicit about 1. defining "streaming" 2. defining "non-streaming" 3. listing all "streaming" SQL operations with links to their reference pages 4. listing "non-streaming" SQL operations with links to their reference pages.  Some of that content is here but I think it should be moved out of here into a separate section called something like "Supported Query Types" and be much more explicit in the form of lists of things.